MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TextFieldFormattingActivity.java

```diff
@@ -0,0 +1,149 @@
+package com.mapbox.mapboxandroiddemo.examples.styles;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Toast;
+
+import com.getbase.floatingactionbutton.FloatingActionButton;
+import com.mapbox.mapboxandroiddemo.R;
+import com.mapbox.mapboxsdk.Mapbox;
+import com.mapbox.mapboxsdk.maps.MapView;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+import com.mapbox.mapboxsdk.maps.OnMapReadyCallback;
+import com.mapbox.mapboxsdk.maps.Style;
+import com.mapbox.mapboxsdk.style.layers.Layer;
+import com.mapbox.mapboxsdk.style.layers.PropertyValue;
+
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.textColor;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.textFont;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.textSize;
+
+/**
+ * Use runtime styling to adjust the font, size, and color of SymbolLayer text fields.
+ */
+public class TextFieldFormattingActivity extends AppCompatActivity {
+
+  private final String[] textFonts = new String[] {"Roboto Black", "Arial Unicode MS Regular"};
+  private final Float[] textSizes = new Float[] {25f, 13f};
+  private final String[] textColors = new String[] {"#00FF08", "#ffd43a"};
+  private MapView mapView;
+  private boolean fontChanged = false;
+  private boolean sizeChanged = false;
+  private boolean colorChanged = false;
```

Comment:
no need to assign false to field declaration, they are already false.

Reply:
fixed